printer_mods/Good handle-Voron/Good handle/README.md

```diff
@@ -0,0 +1,9 @@
+M5X10(SHCS)               4
+
+M5X12(SHCS)               6
+
+M3X20(SHCS)               6
+
+M3X5X5(HEAT SET INSERT)   6
```

Comment:
Is this supposed to the same heatset inserts that are in the BOM? If so, those are M3x5x4.